web3.js/src/connection.ts

```diff
@@ -3707,12 +3721,23 @@ export class Connection {
           // eslint-disable-next-line require-atomic-updates
           sub.subscriptionId = id;
         }
-      } catch (err) {
+      } catch (error) {
         if (sub.subscriptionId === 'subscribing') {
           // eslint-disable-next-line require-atomic-updates
           sub.subscriptionId = null;
         }
-        console.error(`${rpcMethod} error for argument`, rpcArgs, err.message);
+
+        if (Object.values(this._subscribeErrorCallbacks).length) {
+          Object.values(this._subscribeErrorCallbacks).forEach(callback =>
```

Comment:
nit: rather than creating the `Object.values` array twice, can you create it once and reuse it?